packages/@expo/cli/src/run/android/index.ts

```diff
@@ -16,6 +16,7 @@ export const expoRunAndroid: Command = async (argv) => {
     '--no-install': Boolean,
     '--no-bundler': Boolean,
     '--variant': String,
+    '--all-arch': Boolean,
```

Comment:
Let's add a clarification here, or in the help output generation.

```suggestion
    // Unstable, temporary fallback to disable active archs only behavior
    // TODO: replace with better fallback option, like free-form passing gradle props
    '--all-arch': Boolean,
```